contracts/KyberNetwork.sol

```diff
@@ -40,8 +40,9 @@ contract ReentrancyGuard {
 contract KyberNetwork is Withdrawable, Utils2, KyberNetworkInterface, ReentrancyGuard {
 
     enum ReserveType {NONE, PERMISSIONED, PERMISSIONLESS}
-    bytes internal empty;
-    bytes internal permHint = "PERM";
+    bytes internal constant EMPTY_HINT = "";
+    bytes internal constant PERM_HINT = "PERM";
+    uint  public constant PERM_HINT_GET_RATE = 1 << 255; // for get rate. bit mask hint.
```

Comment:
public should probably be before internal.

Reply:
yes
OK
although linter says nonthing.